src/tabs/home/Home.js

```diff
@@ -115,15 +138,23 @@ class Home extends PureComponent {
                               }
                             >
                               <ContentCardConnected
-                                Component={BrandedCard}
+                                Component={CampaignCard}
```

Comment:
Question to make me a better developer: How is `CampaignCard` here getting it's props to the actual component above? I'm trying to prove that being live works, but I'm having issues.

Reply:
Oh wait ... never mind. `BrandedCard` would have worked the same way. I'm a dork.